pxc/jenkins/pxc57-pipeline.groovy

```diff
@@ -51,10 +59,17 @@ pipeline {
             choices: 'yes\nno',
             description: 'Run mysql-test-run.pl',
             name: 'DEFAULT_TESTING')
-        choice(
-            choices: 'no\nyes',
-            description: 'Build with ASAN',
-            name: 'WITH_ASAN')
+        choice(name: 'ANALYZER_OPTS',
+           choices: [
+             '',
+             '-DWITH_ASAN=ON -DWITH_ASAN_SCOPE=ON',
+             '-DWITH_ASAN=ON',
+             '-DWITH_ASAN=ON -DWITH_ASAN_SCOPE=ON -DWITH_UBSAN=ON',
+             '-DWITH_ASAN=ON -DWITH_UBSAN=ON',
+             '-DWITH_UBSAN=ON',
+             '-DWITH_VALGRIND=ON'
+           ].join('\n'),
+           description: 'Analyzer options passed to CMake (ASAN/UBSAN/Valgrind)')
```

Comment:
We should remove whole `parameters` section because it's duplicated with `pxc/jenkins/pxc57-pipeline.yml`.
In 8.0+ we removed `parameters` section many years ago and all works fine.

Reply:
Agree. 
If we're removing duplication, where should we leave the parameters section?
Leaving them in the yml file means that, the job needs to be updated(with jenkins-jobs tool) every time there is a change in any of the parameters. But, if we they're present in the groovy file, there is no need of updating the job, it will automatically picked up by jenkins.